src/Handlebars/Handlebars.php

```diff
@@ -139,6 +147,15 @@ public function __construct(Array $options = [])
         ) {
             $this->aliases = $options['partials_alias'];
         }
+
+        if (isset($options[self::OPTION_ENABLE_DATA_VARIABLES])) {
```

Comment:
Do you mind combining these into a single conditional and then setting $this->enableDataVariables = $options[Handlebars::OPTION_ENABLE_DATA_VARIABLES]; after the conditional check since it will throw an Exception if it's true?